packages/jest-core/src/ReporterDispatcher.ts

```diff
@@ -40,7 +40,6 @@ export default class ReporterDispatcher {
     // Release memory if unused later.
     testResult.sourceMaps = undefined;
     testResult.coverage = undefined;
-    testResult.console = undefined;
```

Comment:
I don't think this is the correct approach, see #8233 (especially the discussion about it breaking reporters)

/cc @scotthovestadt

Reply:
Hmm - would be great to remove console completely from the TestResults type then if it's functionally always set to undefined.

A lot of users of jest-junit want to include console output in the junit.xml output (which is compliant junit) but that's not possible without this change.

Reply:
yeah, we could maybe do a `delete`. Should deffo fix the typings if they are off. And clarify in the docs 🙂 

(this was also probably a breaking change...)